docs/install/collecting-metrics/collector.yaml

```diff
@@ -0,0 +1,40 @@
+# This uses the OpenTelemetry operator:
+# kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
+
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: metrics
+spec:
+  config: |
+    receivers:
+      jaeger:
```

Comment:
Does this post would cover distributing tracing (ie zipkin/jaeger) or that would be a different doc?

Reply:
I think it would in the future, I wanted to go one step at a time.